gradoop-temporal/src/test/java/org/gradoop/temporal/model/impl/layout/TemporalGraphLayoutFactoryTest.java

```diff
@@ -58,7 +58,7 @@
    *
    * @throws Exception if loading the example graph fails
    */
-  @Before
+  @BeforeMethod
```

Comment:
Change to `BeforeClass` and edit the `testFromDataSets()` method to do not manipulate the `vertexDataSet` and `edgeDataSet` instances (better store the result in a local var)